exec/java-exec/src/test/java/org/apache/drill/exec/store/json/TestJsonRecordReader.java

```diff
@@ -179,4 +180,43 @@ public void testNestedFilter() throws Exception {
         .sqlBaselineQuery(baselineQuery)
         .go();
   }
+
+ @Test // See DRILL-4653
+public void testSkippingInvalidJSONRecords() throws Exception {
+    try
+    {
+    String set = "alter session set `" + ExecConstants.JSON_READER_SKIP_INVALID_RECORDS_FLAG+ "` = true";
```

Comment:
these should be indented inside the try block with 2 spaces.   It is best to set the indent level in your IDE (I can help with Eclipse if you are using it;  if you are using IntelliJ I can find out from other developers using IntelliJ).